presto-openapi/src/main/resources/cluster.yaml

```diff
@@ -0,0 +1,139 @@
+openapi: 3.0.0
+info:
+  title: Presto Cluster Stats API
+  description: API for retrieving cluster statistics in Presto.
+  version: 1
+servers:
+  - url: http://localhost:8080
+    description: Presto endpoint when running locally
+paths:
+  /v1/cluster:
+    get:
+      summary: Retrieve cluster statistics
+      description: Retrieves statistics about the Presto cluster, such as running queries, blocked queries, and memory usage.
+      parameters:
+        - name: includeLocalInfoOnly
+          in: query
+          required: false
+          schema:
+            type: boolean
+          description: Include only local information
+          default: false
+      responses:
+        '200':
+          description: Cluster statistics
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ClusterStats'
+  /v1/cluster/memory:
+    get:
+      summary: Retrieve cluster memory pool information
+      description: Retrieves information about the memory pools in the Presto cluster.
+      responses:
+        '200':
+          description: Cluster memory pool information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ClusterMemoryPoolInfo'
```

Comment:
the returned schema is actually a map of `MemoryPoolId` to `MemoryPoolInfo` objects